doc/Changelog.md

```diff
@@ -3,6 +3,11 @@ Changelog {#Changelog}
 
 # git master {master)
 
+* [13](https://github.com/BlueBrain/Monsteer/pull/13):
+  Adapted Monsteer to the new Zeq sessions mechanism introduced in
+  [105](https://github.com/HBPVIS/zeq/pull/105).
+  Adjusted doxygen documentation.
```

Comment:
Hmm, what's wrong with that in your opinion? I though it provides valuable information, no?